### DISCUSSION TITLE
The ability to share / discuss arc models with everyone.

### DISCUSSION THREAD

**Discussion by PetersFozzy** [bow-simulation/virtualbow#257] [Ideas]
I would like to see a way to share arc models with everyone / discuss these models.

**Comment by stfnp** [comment]
Well, the bow files can be shared with other users, but they need to have VirtualBow installed. Some sort of online platform where you can create a bow, copy a link and share it with everyone would be awesome, but I think that's too ambitious for this project right now.

**Comment by PetersFozzy** [comment]
Hello Stefan,
how about just a discussion board where everyone can post their bow for discussion. Download can be done via the cloud (OneDrive, GoogleDrive, etc.).

Hallo Stefan,
wie wäre es denn einfach nur mit einem Diskussionsboard, wo jeder seinen Bogen zur Diskussion stellen kann. Download kann ja über die Wolke erfolgen (OneDrive, GoogleDrive, etc.).

Viele Grüße Peter
Am 11.08.2023 14:10:50 schrieb Stefan Pfeifer ***@***.***>:
Well, the bow files can be shared with other users, but they need to have VirtualBow installed. Some sort of online platform where you can create a bow, copy a link and share it with everyone would be awesome, but I think that's too ambitious for this project right now.
—
Reply to this email directly, view it on GitHub [https://github.com/bow-simulation/virtualbow/discussions/257#discussioncomment-6701041], or unsubscribe [https://github.com/notifications/unsubscribe-auth/AXW66VAKMUIPJ7N24FCZM5TXUYOMNANCNFSM6AAAAAA3KKUMZI].
You are receiving this because you authored the thread.Message ID: ***@***.***>
[2b9f6231-dd9b-4202-b060-48d76e5bebb8]

**Comment by stfnp** [comment]
Hi Peter,

now I see what you mean!

There is a subreddit called [r/virtualbow](https://www.reddit.com/r/VirtualBow/) where anythinig related to VirtualBow can be posted and discussed. This would be a good place to discuss bow models, it's just not very active. Admittedly Reddit might not be everyone's cup of tea though.

But I think any forum dedicated to VirtualBow would probably have the same issue: Not having enough active users, since VirtualBow is still kind of a niche software.

Viele Grüße :)
Stefan
